site/en/guide/autodiff.ipynb

```diff
@@ -120,7 +120,7 @@
       "source": [
         "## Gradient tapes\n",
         "\n",
-        "TensorFlow provides the [tf.GradientTape](https://www.tensorflow.org/api_docs/python/tf/GradientTape) API for automatic differentiation; that is, computing the gradient of a computation with respect to some inputs, usually `tf.Variable`s.\n",
```

Comment:
Please don't remove links as someone can refer from them and it will be easy for others to find the documentation of key terms as `tf.GradientTape` here.

Reply:
Hi @aavishkarmishra Thanks for the feedback! Is this specific to `tf.GradientTape` and should I be aware of any other use cases that don't receive links to API references during the notebook-to-TensorFlow.org preprocessing?

Reply:
@8bitmp3 You are correct, the full symbol is auto-linked when wrapped in backticks. It's fine to switch these, thanks

Reply:
@lamberta Thank you. Regarding the round brackets for methods - let me know how your team wishes to proceed. I've seen use cases of both with and without them (maybe I'm wrong). Again, these are just suggestions so feel free to ignore my commit changes. All feedback welcome!

Reply:
Not covered in our [style guide](https://www.tensorflow.org/community/contribute/code_style), but maybe it should be.
If you use the complete symbol, like `tf.GradientTape.gradient` it will auto-link. It's more important to include when showing params like `foo.mymethod(x, y)`. But if just referring to the method there's no requirement to add parenthesis.